buildscripts/incremental/test.sh

```diff
@@ -41,7 +41,34 @@ else
     esac
 fi
 
+# If TEST_THREADING is set in the env, then check that Numba agrees that the
+# environment can support the requested threading.
+function check_sysinfo() {
+    cmd="import os;\
+         from numba.misc.numba_sysinfo import get_sysinfo;\
+         print(get_sysinfo()['$1 Threading'])"
+    python -c "$cmd"
+}
 
+if [[ "$TEST_THREADING" ]]; then
+    if [[ "$TEST_THREADING" == "tbb" ]]; then
+        tstat=$(check_sysinfo "TBB")
```

Comment:
I find it hard to tell if this is running or not. Can use more `echo` when it passes

Reply:
I've updated the python snippet to print on success in 1b4666b